homeassistant/components/overkiz/sensor.py

```diff
@@ -330,7 +328,6 @@ class OverkizSensorDescription(SensorEntityDescription):
         name="Discrete RSSI Level",
         entity_registry_enabled_default=False,
```

Comment:
Please disable the other RSSI sensor too.

Reply:
Sorry, I forgot to push that change. It is now pushed :-).